libs/layer-composer/src/generators/tile-cluster/tile-cluster.ts

```diff
@@ -32,6 +32,15 @@ class TileClusterGenerator {
         Array.isArray(config.eventTypes) ? config.eventTypes.join(',') : config.eventTypes
       )
     }
+
+    if (config.filters?.flag) {
```

Comment:
What about making this more flexible with something like:
```
Object.values(config.filters).forEach(([filter, value]) => url.searchParams.set(filter, value)
```

Reply:
I tried but the events endpoint expects 'flags' and the filter needs to be called flag to pull the flag list